src/qibo/backends/numpy.py

```diff
@@ -236,16 +232,16 @@ def update_frequencies(nsamples, frequencies):
     def compile(self, func):
         return func
 
-    def device(self, device_name):
-        class DummyModule:
+    class DummyModule:
 
-            def __enter__(self, *args):
-                pass
+        def __enter__(self, *args):
+            pass
 
-            def __exit__(self, *args):
-                pass
+        def __exit__(self, *args):
+            pass
 
-        return DummyModule()
+    def device(self, device_name):
+        return self.DummyModule()
```

Comment:
Maybe we could use instead:
```suggestion
        pass
```

Reply:
Forget, I misread this class.